samples/layouts/tile-manager/layout/src/index.ts

```diff
@@ -0,0 +1,51 @@
+import { defineComponents, IgcTileManagerComponent, IgcButtonComponent, IgcButtonGroupComponent } from 'igniteui-webcomponents';
+import 'igniteui-webcomponents/themes/light/bootstrap.css';
+import './layout.css'
+
+defineComponents(IgcTileManagerComponent, IgcButtonComponent, IgcButtonGroupComponent);
+
+export class TileManagerLayout {
+
+  constructor() {
+    let serializedData: string;
+    var tileManager = document.querySelector<IgcTileManagerComponent>('#tile-manager1')!;
```

Comment:
Define the index for the header and the content
```suggestion
    let index = 4;
    var tileManager = document.querySelector<IgcTileManagerComponent>('#tile-manager1')!;
```